src/components/ButtonWithDropdownMenu/index.tsx

```diff
@@ -118,15 +66,15 @@ function ButtonWithDropdownMenu({
 
     return (
         <View>
-            {options.length > 1 ? (
+            {shouldAlwaysShowDropdownMenu || options.length > 1 ? (
                 <View style={[styles.flexRow, styles.justifyContentBetween, styles.alignItemsCenter, style]}>
                     <Button
-                        success
+                        success={success}
                         pressOnEnter={pressOnEnter}
                         ref={buttonRef}
                         onPress={(event) => onPress(event, selectedItem.value)}
-                        text={selectedItem.text}
-                        isDisabled={isDisabled}
+                        text={customText ?? selectedItem.text}
+                        isDisabled={isDisabled || customText !== undefined}
```

Comment:
NAB the cursor style seems off because of this. I don't think we wanna show the `not-allowed` cursor on this case

Reply:
addressed